### PR TITLE
Show parner fee

### PR DESCRIPTION
**Pull request by anxolin**
# Summary
Continuation on https://github.com/cowprotocol/cowswap/pull/3920 (thanks @shoom3301 for your last 2 commits!)

Fetch the partner fee from the widget params and display it.

Screenshot at Feb 23 18-21-14

Tooltip shows the BPS (and percentage)
Screenshot at Feb 23 18-21-23

## Test
1. Place an order
2. Observe the mocked data (please, for now don't test in production, as we have mocked data)

image

## not included
Computation of the fee is not done

### CODE REVIEW

**Comment by anxolin** [comment]
Consolidating this PR, I still address comments.

File: apps/cowswap-frontend/src/modules/swap/containers/Row/RowFee/index.tsx [hunk 1]
```diff
@@ -86,25 +88,27 @@ export function RowFee({
...
+    const label = swapZeroFee ? 'Est. fees' : 'Fees'
+    const displayFee = realizedFee || feeAmount
     const feeCurrencySymbol = displayFee?.currency.symbol || '-'
     // TODO: delegate formatting to the view layer
-    const smartFeeFiatValue = formatFiatAmount(feeFiatValue)
-    const smartFeeTokenValue = formatTokenAmount(displayFee)
-    const feeAmountWithCurrency = `${smartFeeTokenValue} ${formatSymbol(feeCurrencySymbol)} ${
+    const feeInFiatFormatted = formatFiatAmount(feeInFiat)
+    const displayFeeFormatted = formatTokenAmount(displayFee)
+    const feeAmountWithCurrency = `${displayFeeFormatted} ${formatSymbol(feeCurrencySymbol)} ${
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/swap/containers/Row/RowFee/index.tsx:97]: Is this adding the fee=0 logic back? Didn't know it was removed...

File: apps/cowswap-frontend/src/modules/swap/hooks/useSwapState.tsx [hunk 2]
```diff
@@ -241,17 +242,22 @@ export function useDerivedSwapInfo(): DerivedSwapInfo {
 
   const isWrapping = isWrappingTrade(inputCurrency, outputCurrency, chainId)
 
+  const { partnerFee } = useInjectedWidgetParams()
+
+  console.log('partnerFee', partnerFee)
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/swap/hooks/useSwapState.tsx:247]: Oops

File: apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ export function RowFeeContent(props: RowFeeContentProps) {
     <StyledRowBetween {...styleProps} alignContentRight={showFiatOnly}>
       {!noLabel && (
         <RowFixed>
-          <TextWrapper>{swapZeroFee ? 'Est. fees' : 'Fees'}</TextWrapper>
+          <TextWrapper>{label}</TextWrapper>
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx:40]: Ah, that's where that change is coming from! Disregard the first comment then.

File: apps/cowswap-frontend/src/modules/swap/pure/TradeSummary/index.tsx [hunk 4]
```diff
@@ -16,24 +17,34 @@ const Wrapper = styled.div``
...
             allowsOffchainSigning={allowsOffchainSigning}
             showHelpers={showHelpers}
           />
         )}
 
+        {partnerFee && (
+          <RowPartnerFee
+            showHelpers={true}
+            partnerFee={partnerFee}
+            feeAmount={CurrencyAmount.fromRawAmount(trade.inputAmount.currency, '100000000000000')}
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/swap/pure/TradeSummary/index.tsx:43]: What's this huge magic number?

File: apps/cowswap-frontend/src/modules/swap/containers/Row/RowFee/index.tsx (hunk 1 above)
Comment (anxolin) [reply apps/cowswap-frontend/src/modules/swap/containers/Row/RowFee/index.tsx:97]: It just changes some names. Performed during the refactor of the two fee Rows

File: apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx (hunk 3 above)
Comment (anxolin) [reply apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx:40]: 👍

**Pull request merged**